src/Hubbup.Web/DataSources/DataLoadingService.cs

```diff
@@ -0,0 +1,62 @@
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Hubbup.Web.DataSources;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+
+namespace Hubbup.Web
+{
+    public class DataLoadingService : IHostedService
+    {
+        private static readonly TimeSpan TimerPeriod = TimeSpan.FromMinutes(10);
+
+        private readonly IDataSource _dataSource;
+        private readonly ILogger _logger;
+
+        private readonly Timer _timer;
+        private int _loading = 0;
```

Comment:
I generally dislike implicit defaults, but I know we use them consistently in other stuff so I'll fix it.